### ISSUE TITLE
App still crashing

### ISSUE THREAD

**Issue by mjain234** [Sovan22/Tokeii#20]
I recently updated the app to latest version but when playing any video as you can see [here](https://drive.google.com/file/d/1d7iUMfW_21wPcl5Vw7jx3uiiFkfbwJwT/view), it backs out suddenly. It also happens when I move to another movie/TV show, it crashes around 2-3 times.

**Comment by Sovan22** [comment]
@mjain234 the app in the video is different

**Comment by mjain234** [comment]
Sorry for the issue but check [this](https://drive.google.com/file/d/1d7iUMfW_21wPcl5Vw7jx3uiiFkfbwJwT/view)

**Comment by Sovan22** [comment]
@mjain234 Okay so you are able to watch after running it two three times?

**Comment by mjain234** [comment]
> @mjain234 Okay so you are able to watch after running it two three times?

Yup, why wouldn't I? I am in love with this app for now, i will try to run even after 3 times 😜. But yaah, I was able to watch after that. It happens sometimes. But it can be sometimes frequent  too.

**Comment by Sovan22** [comment]
@mjain234 I will look into these problems. Thanks for bringing them to my notice
